src/content/deadlines/04_photos.mdx

```diff
@@ -1,8 +1,10 @@
 ---
-title: Photos
-subtitle: Refresh you memories!
-url: /photos
+title: 2024 Videos
+subtitle: Upgrade your experience
```

Comment:
Personally, this doesn't sound very good to me. It suggests my experience at the conference was not as good. I prefer "Relive EuroPython 2024"

Reply:
Changed it to your wording.